Audacia.Mail.MailTrap/HostType.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace Audacia.Mail.MailTrap
+{   
+    /// <summary>
+    /// Mailtrap host types enum.    
+    /// </summary>
+    public enum HostType
```

Comment:
Could you swap these round? I'd prefer it if `Test` was the default value, which it will be if it's first.

Reply:
Done